app/pages/app.tsx

```diff
@@ -91,43 +92,43 @@ export const App: FC = ({ children }) => {
   }, [activeStackingTx]);
 
   useEffect(() => {
-    const wsUrl = new URL(activeNode.url);
+    const wsUrl = new URL(api.baseUrl);
     wsUrl.protocol = 'ws:';
     let client: null | StacksApiWebSocketClient;
 
     async function run() {
       client = await connectWebSocketClient(urljoin(wsUrl.toString(), 'extended', 'v1', 'ws'));
       if (!address) return;
       await client.subscribeAddressBalanceUpdates(address, async ({ address, balance }) => {
-        await safeAwait(mutate('mempool'));
+        await safeAwait(mempool.refetch());
         dispatch(updateAddressBalance({ address, balance }));
       });
       await client.subscribeAddressTransactions(address, async ({ tx_id }) => {
-        const newTx = await new Api(activeNode.url).getTxDetails(tx_id);
+        const newTx = await api.getTxDetails(tx_id);
         if (
           isDelegatedStackingTx(newTx.data, poxInfo?.contract_id) ||
           isRevokingDelegationTx(newTx.data, poxInfo?.contract_id) ||
           isDelegateStxTx(newTx.data, poxInfo?.contract_id)
         ) {
-          void updateDelegationStatus();
+          await safeAwait(delegationStatus.refetch());
         }
         if (newTx.data.tx_status !== 'success') return;
         dispatch(addNewTransaction(newTx.data));
         dispatch(pendingTransactionSuccessful(newTx.data));
       });
     }
-
     void run();
     return () => {
       if (client) client.webSocket.close();
     };
-  }, [address, dispatch, activeNode.url, updateDelegationStatus, poxInfo?.contract_id]);
+    // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
Why are you ignoring this? Ideally we'd never

Reply:
Curious for your input. The required additions are `mempool` and `delegationStatus`, just for the refetch methods. But, I don't want to rerun the effect when either of those datasets change, I just want access to the refetch method.

If I add only the refetch method as the dep 👇🏼 

```
React Hook useEffect has missing dependencies: 'delegationStatus' and 'mempool'. Either include them or remove the dependency array.eslintreact-hooks/exhaustive-deps
```